Makefile

```
@@ -31,7 +31,7 @@ test: test-data-gds
 	pytest -s
 
 test-force:
-	run pytest --force-regen -s
+	pytest -n auto --randomly-seed=41 --force-regen
```

```suggestion
	pytest -n auto --force-regen
```

why seed? doesn't that defeat the purpose?

---

Good, catch! i fixed it